packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.test.ts

```diff
@@ -1279,6 +1279,68 @@ describe('user-storage/user-storage-controller - saveInternalAccountToUserStorag
       ),
     ).rejects.toThrow(expect.any(Error));
   });
+
+  it('saves an internal account to user storage when the AccountsController:accountRenamed event is fired', async () => {
+    const arrangeMocks = async () => {
+      return {
+        messengerMocks: mockUserStorageMessenger(),
+      };
+    };
+
+    const { messengerMocks } = await arrangeMocks();
```

Comment:
I like the arrangeMocks pattern (much easier to reason with vs global mocks and beforeEach resetting, etc).

In this case, since the arrangeMocks isn't performing any additional work, can we use `mockUserStorageMessenger` directly?

```ts
const { messenger, baseMessenger } = await mockUserStorageMessenger()
```

Reply:
You're right, definitely cleaner! Updated! :)